src/protocol/templates/ping_protocol.json

```diff
@@ -1,212 +1,511 @@
+            "type": "u16"
+          }
+        ]
+      },
+      "device_id": {
+        "id": "1201",
+        "payload": [],
+        "deprecated": "true"
+      },
+      "voltage_5": {
+        "id": "1202",
+        "payload": [
+          {
+            "name": "mvolts",
+            "type": "u16"
+          }
+        ]
+      },
+      "speed_of_sound": {
+        "id": "1203",
+        "payload": [
+          {
+            "name": "speed_mmps",
+            "type": "u32"
+          }
+        ]
+      },
+      "range": {
+        "id": "1204",
+        "payload": [
+          {
+            "name": "start_mm",
+            "type": "u32"
+          },
+          {
+            "name": "length_mm",
+            "type": "u32"
+          }
+        ]
+      },
+      "mode": {
+        "id": "1205",
+        "payload": [
+          {
+            "name": "auto_manual",
+            "type": "u8"
+          }
+        ]
+      },
+      "ping_rate_msec": {
+        "id": "1206",
+        "payload": [
+          {
+            "name": "msec_per_ping",
+            "type": "u16"
+          }
+        ]
+      },
+      "gain_index": {
+        "id": "1207",
+        "payload": [
+          {
+            "name": "gain_index",
+            "type": "u8"
+          }
+        ]
+      },
+      "pulse_usec": {
+        "id": "1208",
+        "payload": [
+          {
+            "name": "pulse_usec",
+            "type": "u16"
+          }
+        ]
+      },
+      "background_data": {
+        "id": "1209",
+        "payload": [
+          {
+            "name": "gain_index",
+            "type": "u8"
           }
+        ]
+      },
+      "general_info": {
+        "id": "1210",
+        "payload": [
+          {
+            "name": "gain_index",
+            "type": "u8"
+          }
+        ]
+      },
+      "distance_simple": {
+        "id": "1211",
+        "payload": [
+          {
+            "name": "distance",
+            "type": "u32"
+          },
+          {
+            "name": "confidence",
+            "type": "u8"
+          }
+        ]
+      },
+      "distance": {
+        "id": "1212",
+        "payload": [
+          {
+            "name": "distance",
+            "type": "u32"
+          },
+          {
+            "name": "confidence",
+            "type": "u8"
+          },
+          {
+            "name": "pulse_usec",
+            "type": "u16"
+          },
+          {
+            "name": "ping_number",
+            "type": "u32"
+          },
+          {
+            "name": "start_mm",
+            "type": "u32"
+          },
+          {
+            "name": "length_mm",
+            "type": "u32"
+          },
+          {
+            "name": "gain_index",
+            "type": "u32"
+          }
+        ]
+      },
+      "processor_degC": {
+        "id": "1213",
+        "payload": [
+          {
+            "name": "temp",
+            "type": "u16",
+            "units": "centiC",
```

Comment:
centi is value/100, value*100 is hecto or just h.
I think that the unit should be hC, stands for hecto Celsius

Reply:
the value transmitted is centiC, check again.